src/solvers/dgmulti/sbp.jl

```diff
@@ -395,3 +404,74 @@ function calc_surface_integral!(du, u, surface_integral::SurfaceIntegralWeakForm
   @assert nelements(mesh, dg, cache) == 1
   nothing
 end
+
+function create_cache(mesh::DGMultiMesh, equations, dg::DGMultiFluxDiffPeriodicFDSBP, RealT, uEltype)
+
+  rd = dg.basis
+  md = mesh.md
+
+  # for use with flux differencing schemes
+  Qrst_skew = compute_flux_differencing_SBP_matrices(dg)
+
+  # storage for volume quadrature values, face quadrature values, flux values
+  nvars = nvariables(equations)
+  u_values = allocate_nested_array(uEltype, nvars, size(md.xq), dg)
+
+  # dummy storage to allow for compatibility with DGMultiFluxDiffSBP
+  fluxdiff_local_threaded = [zeros(SVector{nvars, uEltype}, size(md.xq, 1))]
+
+  return (; md, Qrst_skew, u_values, fluxdiff_local_threaded,
+            invJ = inv.(md.J), inv_wq = inv.(rd.wq),)
```

Comment:
As usual, I am a big fan of self-descriptive names, i.e., `inverse_jacobian` and `inverse_wq` (or `inverse_whatevergoodnamethereis`) 😬

Reply:
I'd prefer to save this for another PR, just b/c it touches pretty much every single `DGMulti` solver 😅.

Reply:
Haha, ok, no worries!